portal-impl/src/com/liferay/portal/image/ImageToolImpl.java

```diff
@@ -734,7 +734,9 @@ protected RenderedImage doScale(
 
 		int type = originalBufferedImage.getType();
 
-		if (type == BufferedImage.TYPE_CUSTOM) {
+		if ((type == BufferedImage.TYPE_CUSTOM) ||
```

Comment:
the if with the check of the  : _BufferedImage.TYPE_BYTE_INDEXED_ and _BufferedImage.TYPE_CUSTOM_ maybe should ordered by letter.

if the order is significant maybe must tell Brian about it or separate on 2 ifs

Reply:
Hi @AliciaGarciaGarcia,
the type has one integer value, so it matches one bufferedImage type or neither, so the order is not significant.